### DISCUSSION TITLE
Preview service not switches to the latest image pod

### DISCUSSION THREAD

**Discussion by sampathkaran** [argoproj/argo-rollouts#3571] [General]
Hi,
I tried to install argo rollout. When I try to upgrade the version of image , I could see both the active service and preview points to the older version of the image. I tried checking rollout logs but not able to figure it out. Can someone help ?

**Comment by kostis-codefresh** [comment]
Can you post the rollout of your application?

And the output of the argo Rollouts CLI during the deployment?
